workflow/controller/controller.go

```diff
@@ -259,27 +259,31 @@ func (wfc *WorkflowController) periodicWorkflowGarbageCollector(stopCh <-chan st
 		case <-ticker.C:
 			if wfc.offloadNodeStatusRepo.IsEnabled() {
 				log.Info("Performing periodic workflow GC")
-				oldUIDs, err := wfc.offloadNodeStatusRepo.ListOldUIDs(wfc.GetManagedNamespace())
+				oldRecords, err := wfc.offloadNodeStatusRepo.ListOldOffloads(wfc.GetManagedNamespace())
 				if err != nil {
-					log.WithField("err", err).Error("Failed to list offloaded nodes")
+					log.WithField("err", err).Error("Failed to list old offloaded nodes")
 				}
-				if len(oldUIDs) == 0 {
-					log.Info("Zero old UIDs, nothing to do")
+				if len(oldRecords) == 0 {
+					log.Info("Zero old records, nothing to do")
```

Comment:
minor,  Can you reverse the logging if there are records? 
```
if len(oldRecords) > 0 {
```
which will reduce the logging